### ISSUE TITLE
is this plugin support strapi v4.0.0 ?

### ISSUE THREAD

**Issue by GaiheiluKamei** [meilisearch/strapi-plugin-meilisearch#331]
**Description**
When i use strapi v3.6.8, with the following command 
```
npm i strapi-plugin-meilisearch
npm run build
docker run -it --rm -p 7700:7700 getmeili/meilisearch:latest ./meilisearch
npm run develop
```

Then strapi dashboard Plugin part appear **Meilisearch**, now i use strapi v4.0.0 with the same command, dashboard doesnot show "Meilisearch" Plugin anymore.

**Comment by bidoubiwa** [comment]
Hello! The migration to v4 is on its way, see this repo https://github.com/meilisearch/strapi-plugin-meilisearch-v4. 
You can follow the completion here: 
#271

**Comment by GaiheiluKamei** [comment]
> Hello! The migration to v4 is on its way, see this repo https://github.com/meilisearch/strapi-plugin-meilisearch-v4. You can follow the completion here: #271

Thank you so much for reply, seems this plugin have not support strapi v4 fully, in this case i will down my strapi version to v3.6.8.

**Comment by bidoubiwa** [comment]
closed in favor of #271
